bokeh/models/tools.py

```diff
@@ -1187,6 +1187,10 @@ class HoverTool(InspectTool):
     Whether tooltip's arrow should be shown.
     """)
 
+    tooltip_arrow_color = Color(default="#909599", help="""
+    The color of the tooltip's arrow as a CSS colour (e.g. '#FFFFFF', 'white', etc.).
+    """)  # XXX: keep default in sync with tooltips.less
+
```

Comment:
It's a bit awkward having this property both here and on the tooltip. Ideally this would be configurable via
```
hover_tool.tooltip.arrow_fill_color = "red"
```
but we are not quite set up to do that yet. That would be part of the the "sticky tooltip" work that has never been accomplished. I wonder if it would be better to wait on this feature until Tooltip becomes more of a first-class obejct that can be configured directly.

Reply:
ref for sticky tooltip: https://github.com/bokeh/bokeh/issues/5724

Reply:
Yeah it definitely felt weird copy/pasting the properties around, and the proposed syntax looks cleaner for the user too. 

I'll still complete the requested changes in this PR in case you change your mind or at least to leave it as a reference for a future implementation.